spatialprofilingtoolbox/db/scripts/upload_sync_small.py

```diff
@@ -0,0 +1,85 @@
+"""Synchronize a small data artifact with the database."""
```

Comment:
Next time, consider using e.g.

```
git mv upload_sync_findings.py upload_sync_small.py
```

to rename the file before making your changes. Git registered the whole file as new, erasing the history.

Reply:
Gotcha, thanks for doing that.